app/townsquare/models.py

```diff
@@ -394,9 +394,14 @@ def __str__(self):
 class Favorite(SuperModel):
     """Model for each favorite."""
     user = models.ForeignKey(get_user_model(), on_delete=models.CASCADE, related_name='favorites')
-    activity = models.ForeignKey('dashboard.Activity', on_delete=models.CASCADE)
+    grant = models.ForeignKey('grants.Grant', on_delete=models.CASCADE, related_name='grant_favorites', null=True)
+    activity = models.ForeignKey('dashboard.Activity', on_delete=models.CASCADE, null=True)
     created = models.DateTimeField(auto_now=True)
 
+    @staticmethod
+    def grants():
+        return Favorite.objects.filter(activity=None)
```

Comment:
why use the `activity=None` filter to find grants instead of `Favorite.objects.filter(grant__isnull=False)`? Seems unintuitive

Reply:
Also the related_name should probably just be `'favorites'` so you can access it like `grant.favorites`